packages/pwa/app/pages/account/wishlist/index.jsx

```diff
@@ -124,25 +114,15 @@ const AccountWishlist = () => {
                     buttonProps={{leftIcon: undefined}}
                     onButtonClick={() => navigate('/')}
                 />
-            </Stack>
-        )
-    }
-
-    return (
-        <Stack spacing={4} data-testid="account-wishlist-page">
-            <Heading as="h1" fontSize="2xl">
-                <FormattedMessage defaultMessage="Wishlist" />
-            </Heading>
+            )}
 
-            {wishlist._productItemsDetail &&
-                wishlist?.customerProductListItems.map((item) => (
+            {isInitialized &&
+                !isEmpty &&
+                wishlist?.data?.customerProductListItems?.map((item) => (
```

Comment:
do we need to guard the data here taking into the account that we know the list is not empty at this point?

Reply:
good call, probably not! i fixed it